test/Nancy.Tests/Unit/Sessions/DefaultSessionObjectFormatterFixture.cs

```diff
@@ -68,11 +72,16 @@ public void Should_return_null_when_deserializing_empty_string()
         [Serializable]
         public class Payload : IEquatable<Payload>
         {
-            public int IntValue { get; private set; }
+            public int IntValue { get;  set; }
+
+            public bool BoolValue { get;  set; }
 
-            public bool BoolValue { get; private set; }
+            public string StringValue { get;  set; }
 
-            public string StringValue { get; private set; }
+            public Payload()

```

Comment:
> Sure, it's a breaking change, but i think it's an acceptable one.

I'm not arguing that. It's just that we're making breaking changes left and right without marking them as such, or without even realizing that we're breaking stuff.